File: Arrays and String/119. Pascal's Triangle II/pascal.cpp
class Solution
{
public:
    vector<int> getRow(int rowIndex)
    {
        vector<int> res;
        res.push_back(1);
        long long prev = 1;

        for (int k = 1; k <= rowIndex; k++)
        {
            long long val = prev * (rowIndex - k + 1) / k;
            res.push_back(val);
            prev = val;
        }
        return res;
    }
};

File: 3. Arrays and String/119. Pascal's Triangle II/pascal.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        res = [1]
        prev = 1

        for k in range(1, rowIndex + 1):
            val = prev * (rowIndex - k + 1) // k
            res.append(val)
            prev = val

        return res